app/services/action_runner.py

```diff
@@ -16,13 +16,12 @@
 logger = logging.getLogger(__name__)
 
 
-async def execute_action(integration_id: str, action_id: str, config_overrides: dict = None, config_data: dict = None):
```

Comment:
Agree. I'm in favor of using the existent `config_overrides` instead of adding a new parameter.